Forms/MainForm.cs

```diff
@@ -361,13 +410,30 @@ private void txtCharacterList_TextChanged(object sender, EventArgs e)
             CharacterList charList = new CharacterList();
             charList.AddRange(txtCharacterList.Text.ToArray());
             Configuration.DefaultCharacters = charList;
+            LoadConfigurationDetails();
         }
 
         private void txtWordList_TextChanged(object sender, EventArgs e)
         {
+            int oldWordCount = Configuration.DefaultWords.Count;

```

Comment:
We can add copy-paste keypress and yes probably have a timer to "async" update after keypresses.
I like that better as opposed to parsing for spaces twice, i think this can become heavy with bigger wordlist.